IBAnimatable/GradientDesignable.swift

```diff
@@ -104,3 +106,48 @@ private class PrivateGradientView: UIView {
   }
 
 }
+
+extension CAGradientLayer {
+  fileprivate func configurePoints(with startPoint: GradientStartPoint) {
+    switch startPoint {
+    case .top:
+      self.startPoint = CGPoint(x: 0.5, y: 0)
+      self.endPoint = CGPoint(x: 0.5, y: 1)
+    case .topRight:
+      self.startPoint = CGPoint(x: 1, y: 0)
+      self.endPoint = CGPoint(x: 0, y: 1)
+    case .right:
+      self.startPoint = CGPoint(x: 1, y: 0.5)
+      self.endPoint = CGPoint(x: 0, y: 0.5)
+    case .bottomRight:
+      self.startPoint = CGPoint(x: 1, y: 1)
+      self.endPoint = CGPoint(x: 0, y: 0)
+    case .bottom:
+      self.startPoint = CGPoint(x: 0.5, y: 1)
+      self.endPoint = CGPoint(x: 0.5, y: 0)
+    case .bottomLeft:
+      self.startPoint = CGPoint(x: 0, y: 1)
+      self.endPoint = CGPoint(x: 1, y: 0)
+    case .left:
+      self.startPoint = CGPoint(x: 0, y: 0.5)
+      self.endPoint = CGPoint(x: 1, y: 0.5)
+    case .topLeft:
+      self.startPoint = CGPoint(x: 0, y: 0)
+      self.endPoint = CGPoint(x: 1, y: 1)
+    case let .custom(start, end):
+      self.startPoint = CGPoint(x: start.x, y: start.y)
+      self.endPoint = CGPoint(x: end.x, y: end.y)
+    case .none:
+      break
+    }
+  }
+
+  func image() -> UIImage? {
```

Comment:
should call `makeImage()` or use `var image: UIImage?` . `makeImage()` means we create a new UIImage instance. And `var image: UIImage?` means `image` is a property of `CAGradientLayer`. I more prefer `makeImage` since this image is a copy of the `CAGradientLayer` not a property of it. How do you think?